gulp/DevelopmentTasks.ts

```diff
@@ -145,7 +145,14 @@ const doc: ITaskFunction = async () => {
             commandNameSummary += " (experimental)";
         }
 
-        const anchorTag = "module-" + definition.name;
+        // create anchor tag that matches full command
+        var anchorTag = "";
+        if (oldCommandName !== "") {
+            anchorTag = oldCommandName + "-" + definition.name;
```

Comment:
This is optional really but you could use template literals here
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
so it could be 
```
anchorTag = `${oldCommandName}-${definition.name}`;
```

Reply:
Changed to template literals.